web/src/routes/(user)/albums/+page.svelte

```diff
@@ -1,11 +1,20 @@
+<script lang="ts" context="module">
+  // table is the text printed in the table and order is the text printed in the dropDow menu
+  export interface Sort {
```

Comment:
Maybe this interface can `sortFn: ` that is a method that takes in a list of albums and returns them sorted in descending order. Then you can just call that and optionally reverse it.

Another option is to add a function that you call on a single album that returns a string value that you can sort on. Then the main sort implementation is like `albums.sort((a, b) =>mapFn(a).localCompare(mapFn(b))`

Reply:
> Maybe this interface can `sortFn: ` that is a method that takes in a list of albums and returns them sorted in descending order. Then you can just call that and optionally reverse it.

I don't fully understand; What is the list sorted by ?

Reply:
Each option in the `sortByOptions` can have its own implementation to sort a list of albums. So it has a title, label, direction, and a sort function.

Reply:
Yes, makes sense. Thanks !

Reply:
Looks a lot better. I think we still are using lodash-es. Might be possible to use that order by method too for more concise code